webapp-django/crashstats/settings/base.py

```diff
@@ -757,7 +757,7 @@ def path(*dirs):
     'https://www.google-analytics.com',
     'data:',  # what jquery.tablesorter.js's CSS uses
 )
-CSP_CHILD_SRC = (
+CSP_FRAME_SRC = (
```

Comment:
`child-src` is deprecated and being replaced with `frame-src` and `worker-src`. @peterbe had a conversation with someone at some point about this and we left it as `child-src`. I couldn't find that conversation, though. Anyhow, so I'm switching it to `frame-src` figuring that conversation happened a while ago and it's probably fine to switch now.

Reply:
I can't remember which is which but the predicament was that if we upgraded django-csp to the latest, it would complain about one of them not being set and that you should set the other (e.g. use child-src, not frame-src (or the other way around)). I checked with Greg and it was correct. 
The problem is that even with Nightly (at the time) you would get a CSP warning every single time in the web console because the new CSP spec hasn't made it to all browsers yet. 
So I decided to hold off a wee bit until Nightly would get that fixed so you wouldn't have to have a warning in the web console every single time.

Reply:
Ahhh... ok. We updated django-csp, so we don't have that problem anymore. And according to MDN, frame-src is supported by most browsers.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-src#Browser_compatibility

So I think we're good now. Thank you, @peterbe!